bq-connector/docai_bq_connector/connector/BqDocumentMapper.py

```diff
@@ -59,7 +65,13 @@ def __init__(
         self.dictionary = self._map_document_to_bigquery_schema(self.fields, bq_schema)
 
     def _parse_document(self) -> List[DocumentField]:
-        row = self._parse_entities(self.processed_document.document.entities)
+        row: DocumentRow
+        if self.parsing_methodology == PARSING_METHOD_ENTITIES:
+            row = self._parse_entities(self.processed_document.document.entities)
+        elif self.parsing_methodology == PARSING_METHOD_FORM:
+            row = self._parse_form_entities(self.processed_document.document)
+        else:
```

Comment:
Shouldn't you also have a case for `PARSING_METHOD_NORMALIZED_VALUES` ?

Reply:
PARSING_METHOD_NORMALIZED_VALUES isn't fully implemented yet. I haven't come across a document yet between the two use-cases I ran that have those fields populated. I can remove it, or we can complete it once we have a better working example.